test/amauri.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('test amauri', () => {
+  it('should be amauri', () => {
+    let flag = 'Amauri';
+    flag = flag.toLowerCase();
+    expect(flag).to.deep.equal('amauri');
+  });
+
+  it('should be amauri fail', () => {
+    let flag = 'Amauri';
+    flag = flag.toLowerCase();
+    expect(flag).to.deep.equal('AMAURI');
```

Comment:
isso aqui deveria quebrar

Reply:
sim, ele quebra

Reply:
foi pra testar que quebrava msm

Reply:
Nesse caso, o teste tem que ser com negação `not.equal`.

Fora isso, é indicado não usar [arrow functions](https://mochajs.org/#arrow-functions) nas callbacks de `describe`, `it`, `before`, etc (no caso, eu evito usar, mesmo que não dê nenhum problema aparente).

Reply:
fixed

Reply:
Altera o CI pra rodar esses testes que vc fez (e lint, se tiver).
No caso, ele teria quebrado anteriormente e seria bem mais fácil de identificar.

Reply:
Ivanildo falou que não é necessário a configuração do Circle CI por enquanto